src/playlist-maker.js

```diff
@@ -281,7 +295,25 @@ const factory = (player, initialList, initialIndex = 0) => {
     autoadvance(delay) {
       playlist.autoadvance_.delay = delay;
       autoadvance.setup(playlist.player_, delay);
+    },
+
+    /**
+     * Sets `repeat` option, which makes the "next" video of the last video
+     * is the first one.
+     *
+     * @param {Boolean} val
+     */
+    repeat(val) {
+      if (val !== undefined) {
+        if (typeof val !== 'boolean') {
+          videojs.log.error('Invalid value for repeat: "' + val + '"');
```

Comment:
`val` should probably be passed as a separate argument to `log.error`, it should end up being printed correctly in the console then and not just `toString`ed.